### ISSUE TITLE
Which loss did you? dense or sparse?

### ISSUE THREAD

**Issue by YanShuo1992** [eric-yyjau/pytorch-superpoint#42]
Hi, there

I find it is confusing that there are dense loss and sparse loss.  The config you provided used dense loss function, while the original paper used sparse loss.

Which one did you use for the provided weight?

By the way, what does heat2.0 mean in model name?

Cheers

**Comment by eric-yyjau** [comment]
Hi @YanShuo1992,

Thank you for your question.
I implemented dense loss then sparse loss. They have similar performance in my experiments.
The COCO model used sparse loss (https://github.com/eric-yyjau/pytorch-superpoint/blob/4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691/logs/superpoint_coco_heat2_0/config.yml#L84).

For `heat`, I named when I was experimenting adding gaussian kernel to the labels, i.e. heatmap.
(https://github.com/eric-yyjau/pytorch-superpoint/blob/4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691/logs/superpoint_coco_heat2_0/config.yml#L40)

Hope this helps.
Best,
